conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/bitcoin/BitcoinIndexer.scala

```diff
@@ -3,25 +3,25 @@ package tech.cryptonomic.conseil.indexer.bitcoin
 import java.util.concurrent.Executors
 
 import scala.concurrent.{ExecutionContext, Future}
-import scala.concurrent.duration.FiniteDuration
 
 import cats.effect.{IO, Resource}
-import org.http4s.headers.Authorization
-import org.http4s.BasicCredentials
-import org.http4s.client.blaze.BlazeClientBuilder
+// import org.http4s.headers.Authorization
```

Comment:
either uncomment or remove those lines - I suspect if it's commented it is not needed.

Reply:
ok, uncommented and fixed